src/Microsoft.IdentityModel.Tokens/Experimental/Validation/Validators.Issuer.cs

```diff
@@ -52,18 +60,26 @@ await validationParameters.IssuerValidatorAsync(
                         cancellationToken).ConfigureAwait(false);
 
                 if (!result.Succeeded)
-                    return result.Error!.AddCurrentStackFrame();
+                {
+                    if (!ValidationError.TryGetStackFrame(out stackFrame))
+                        stackFrame = ValidationError.GetAsyncStackFrame(new StackFrame(0, true));
+
+                    return result.Error!.AddStackFrame(stackFrame ?? new StackFrame(0, true));
```

Comment:
`stackFrame` is never null in these cases - either `TryGetStackFrame` or `GetAsyncStackFrame` returns one frame.

Reply:
@pmaytak you are correct, i understand we can mark the class that if Succeeded is false then Error cannot be null.
Do you know how to do that?